src/serializer/serializer.js

```diff
@@ -91,6 +93,25 @@ export class Serializer {
     return code;
   }
 
+  processOutputEntries(): boolean {
```

Comment:
This name is very generic, so a comment giving a brief idea of what it is about will be very helpful to a future reader.